src/entity/GeoObjectHandler.ts

```diff
@@ -122,30 +122,51 @@ export class GeoObjectHandler {
         this.update();
     }
 
-    public setColorTextureTag(src: string, tag: string) {
+    public setColorTextureTag(src: string | HTMLImageElement, tag: string) {
         const tagData = this._instanceDataMap.get(tag);
         if (tagData) {
-            tagData._colorTextureSrc = src;
+            if (typeof src === 'string') {
+                tagData._colorTextureSrc = src;
+                tagData._colorTextureImage = null;
+            }
+            if (src instanceof HTMLImageElement) {
+                tagData._colorTextureSrc = null;
+                tagData._colorTextureImage = src;
+            }
             this._instanceDataMap.set(tag, tagData);
-            this._loadColorTexture(tagData);
+            tagData.loadColorTexture();
         }
     }
 
-    public setNormalTextureTag(src: string, tag: string) {
+    public setNormalTextureTag(src: string | HTMLImageElement, tag: string) {
         const tagData = this._instanceDataMap.get(tag);
         if (tagData) {
-            tagData._normalTextureSrc = src;
+            if (typeof src === 'string') {
```

Comment:
Will todo, next time, we will discuss, and not forget, just leave it here
tagData.setNormalTexture(src), and replace all into tagData.